integr8ly/tutorial-web-app
# [WIP] INTLY-476: Update PF4 Branch with latest from Master

## Motivation
Update the PF4 branch with the latest from Master, in order to begin Milestone 2 work for converting to PatternFly 4.

## What
Rebase Master onto PF4

## Why
Part of the PatternFly 4 Milestone work.

## Verification Steps

1. Run `yarn install` to get the latest dependencies
2. Run `yarn run test` to verify tests pass
3. Run `yarn build; yarn start:dev` to start the webapp

## Checklist:


## Progress
